scss/forms/_quantity-selector.scss

```diff
@@ -25,28 +25,55 @@
   }
 
   button {
+    margin-left: 0 !important; // stylelint-disable-line declaration-no-important
```

Comment:
I'm wondering if instead of having `.input-group` on the QS, we could update the CSS to have `display: flex` on `.quantity-selector` and `.quantity-selector > * {flex-shrink: 0;}`. I think it would avoid having this `!important` and resolve a small shift we have at the moment.